flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp

```diff
@@ -135,20 +140,37 @@
     <aura:attribute name="hideShow" type="String" default="show"/>
 
     <aura:attribute name="keyField" type="String" default="Id"/>
+    <aura:attribute name="buttons" type="Boolean" default="false"/>
```

Comment:
A good practice with boolean attributes is to name them to try and make it clear what the states are. for example isActivated is a better boolean name than status. It's not quite clear to me as a reader what this boolean represents: showButtons? useButtons?